src/stdune/scanf.ml

```diff
@@ -0,0 +1,14 @@
+module Scanf = Dune_caml.Scanf
+
+let unescaped x =
+  match Scanf.unescaped x with
+  | exception Scanf.Scan_failure _ -> Error ()
+  | x -> Ok x
+
+let sscanf x fmt f =
+  let module M = struct
+    exception E
```

Comment:
We should put this at the toplevel given that defining an exception is not cost-free. Indeed it requires generating a global identifier.